app/subscriber/src/features/home/Landing.tsx

```diff
@@ -0,0 +1,73 @@
+import { MenuItemNames } from 'components/layout/constants/MenuItemNames';
```

Comment:
This will undergo some more separation of components when work begins on corresponding stories.